en/docs/guides/identity-lifecycles/enable-account.md

```diff
@@ -1,93 +1,196 @@
+4.  Navigate to **Main** > **Identity** > **Users and Roles** > **List** > **Users** and click on
+    **User Profile** of the user account that you want to disable.
+5.  Tick the **Account Disabled** checkbox and click **Update**.
+
+!!! tip
+    
+    **Alternatively,** instead of using the management console, you can also
+    enable/disable the user account using the
+    `         setUserClaimValues        ` method in the
+    `                   RemoteUserStoreManagerService                 `
+    after you have configured WSO2 IS for account disabling. For more information on using admin services, refer [Call Admin Services](../../../apis/call-admin-services)
+    
+
+### Send email notifications for account disabling
+
+Once you have configured WSO2 Identity Server for account disabling, you
+can also configure the WSO2 IS to send an email to the user's email
+address when the user account is disabled. To configure this, follow the
+steps below.  
+
+{! fragments/configure-email-sending.md !}
+
+    !!! info
+        The email template used to send the email notification for
+        account locking is the **AccountLock** template and the template
+        used for account disabling is the **AccountDisable** template. You
+        can edit and customize the email template. For more information on
+        how to do this, see [Customize Automated Emails](../../../guides/tenants/customize-automated-mails)
+.
 
 ---
 
 ## Disable user accounts using SCIM
 
-1. Open the `<IS-HOME>/repository/conf/scim2-schema-extension.config` file and add the following configuration. 
-
-   ```
-   {
-      "attributeURI":"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:accountDisable",
-      "attributeName":"accountDisable",
-      "dataType":"boolean",
-      "multiValued":"false",
-      "description":"Disable user account.",
-      "required":"false",
-      "caseExact":"false",
-      "mutability":"readwrite",
-      "returned":"default",
-      "uniqueness":"none",
-      "subAttributes":"null",
-      "canonicalValues":[],
-      "referenceTypes":[]
-   }
-   ```
-
-2. Add the `accountDisable` attribute as a sub-attribute of User.
-
-   ```
-   "subAttributes":"verifyEmail askPassword accountDisable employeeNumber costCenter organization division department manager"
-   ```
+1.	Make sure the following configurations are present in the `<IS-HOME>/repository/conf/scim2-schema-extension.config` file.
+
+	```
+	{
+	"attributeURI":"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:accountDisabled",
+	"attributeName":"accountDisabled",
+	"dataType":"boolean",
+	"multiValued":"false",
+	"description":"Store whether the user account is disabled or not",
+	"required":"false",
+	"caseExact":"false",
+	"mutability":"readWrite",
+	"returned":"default",
+	"uniqueness":"none",
+	"subAttributes":"null",
+	"canonicalValues":[],
+	"referenceTypes":[]
+	}
+	```
+
+2.	Add the `accountDisable` attribute as a sub-attribute of User.
```

Comment:
this is not needed as it is there by default

Reply:
By default, the sub attribute value is null. Don't you think this should be mentioned?